### PR TITLE
Use v2 module for 2.0.0 release

### PR DESCRIPTION
**Pull request by justing-bq**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by byronlin13** [comment]
We must cherry pick out the v2 changes in the non-v2 folder

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ First, ensure your project is using [Go modules](https://blog.golang.org/using-g
 
 In your project directory, install the driver using go get:
 
-```go get github.com/awslabs/amazon-qldb-driver-go/qldbdriver```
+```go get github.com/awslabs/amazon-qldb-driver-go/qldbdriver/v2```
```
Comment (byronlin13) [inline README.md:31]: I think we still need to support v1 qldbdriver aswell.

File: go.mod [hunk 2]
```diff
@@ -3,14 +3,9 @@ module github.com/awslabs/amazon-qldb-driver-go
 go 1.14
 
 require (
-	github.com/amzn/ion-go v1.1.3
```
Comment (byronlin13) [inline go.mod:6]: Why did we remove this?

Comment (justing-bq) [reply go.mod:6]: Those dependencies are not necessary for that particular `go.mod` file. But they do exist in `v2/go.mod`.

**Comment by byronlin13** [comment]
Build is failing

File: go.mod (hunk 2 above)
Comment (byronlin13) [reply go.mod:6]: Why are they not necessary?

File: .github/workflows/go.yml [hunk 3]
```diff
@@ -34,6 +34,9 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2.0.0
 
+    - name: Enter v2 directory
```
Comment (byronlin13) [inline .github/workflows/go.yml:37]: we need to test v1 aswell

File: README.md (hunk 1 above)
Comment (justing-bq) [reply README.md:31]: v1 is no longer on the master branch.

File: go.mod (hunk 2 above)
Comment (justing-bq) [reply go.mod]: Turns out this entire file is unnecessary (and also responsible for breaking the CodeQL analysis) so it's removed.

File: .github/workflows/go.yml (hunk 3 above)
Comment (justing-bq) [reply .github/workflows/go.yml]: v1 is on its own branch rather than master.

**Comment by justing-bq** [comment]
> Build is failing

Build is passing now.

**Comment by justing-bq** [comment]
Closing in favor of a future PR.

**Pull request closed without merging**